.github/ISSUE_TEMPLATE/00_BB_issue_template.yml

```diff
@@ -0,0 +1,59 @@
+name: BB bug
+description: File an issue for a BB.
+title: "[BB Bug]:"
+labels: [BB, bug]
+
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thanks for taking the time to fill out this form!
+
+- type: input
+  id: contact
+  attributes:
+    label: Contact Details
```

Comment:
This would normally happen directly in the form of a discussion in the project issue tracker - no separate contact information required. 
(also, I wouldn't put any in, if I had to fill in this form  -  that's what the github communication channels are for)

Reply:
We would prefer to get real names and affiliations to have alternate ways of contacting. That's why we want to keep the contact form.

Reply:
yes with the .yml files we can customise the Github Issue function. See: https://github.com/CSA-FEDERATE/Proposed-BuildingBlocks/issues/new/choose

Reply:
yes with the .yml files we can customise the Github Issue function. See: https://github.com/CSA-FEDERATE/Proposed-BuildingBlocks/issues/new/choose